enterprise/ha/src/test/java/org/neo4j/ha/TestPullUpdatesApplied.java

```diff
@@ -152,7 +153,9 @@ public void failed( InstanceId server )
         Thread.sleep( 15000 );
 
         restart( toKill ); // recovery and branching.
-        boolean hasBranchedData = new File( targetDirectory, "branched" ).listFiles().length > 0;
+        File[] branchedFiles = new File( targetDirectory, "branched" ).listFiles();
+        assertNotNull( branchedFiles );
```

Comment:
`branchedFiles` could be null when `File( targetDirectory, "branched" )` does not exist. Thus `assertNotNull()` looks a bit strange.